tests/Doctrine/Tests/ORM/Functional/Ticket/DDC2632Test.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Tools\Export\Driver\YamlExporter;
+use Doctrine\ORM\Tools\Export\Driver\XmlExporter;
+use Doctrine\Tests\ORM\Functional\DatabaseDriverTestCase;
+/**
+ * @group DDC-2632
+ */
+class DDC2632Test extends DatabaseDriverTestCase

```

Comment:
The test mixes exporter and database driver tests. The exporter tests should work without a real DB (just metadata objects), and the database driver test should just retrieve the metadata.